src/System.Reflection.Metadata.Cil/tests/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name":"System.Reflection.Metadata.Cil.Tests",
+    "dependencies": {
+        "Microsoft.NETCore.TestHost": "1.0.0-beta-23409",
+        "Microsoft.NETCore.Platforms": "1.0.1-beta-23409",
+        "Microsoft.NETCore.Runtime.CoreCLR": "1.0.1-beta-23409",
+        "System.Console": "4.0.0-beta-23516",
+        "System.IO.FileSystem": "4.0.0",
+        "src": "1.0.0",
```

Comment:
I added the type to it:

{
          "type": "build",
          "version": "1.0.0"
        }